server/src/main/java/org/eclipse/openvsx/json/NamespaceMembershipJson.java

```diff
@@ -0,0 +1,23 @@
+/********************************************************************************
+ * Copyright (c) 2020 TypeFox and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ********************************************************************************/
+package org.eclipse.openvsx.json;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;;
+
+@JsonInclude(Include.NON_NULL)
+public class NamespaceMembershipJson extends UserJson {
+    public long id;
```

Comment:
The database entry id is not necessary in the client and should not be exposed here.